frontend/src/layers/ThermalVelocity.tsx

```diff
@@ -11,6 +13,15 @@ export class ThermalVelocity {
     ctx.fillStyle = `rgba(${color.red}, ${color.green}, ${color.blue}, 0.25)`;
     ctx.fillRect(topLeft.x, topLeft.y, bottomRight.x - topLeft.x, bottomRight.y - topLeft.y);
   }
+
+  summary(forecastPoint: ForecastPoint): JSX.Element {
+    return <table>
+      <tbody>
+        <tr><th>Thermals velocity: </th><td>{ forecastPoint.thermalVelocity }&nbsp;m/s</td></tr>
```

Comment:
no "s" on "Thermals velocity" ?

Reply:
Thank you, fixed in [d0d7bad](https://github.com/soaringmeteo/soaringmeteo/pull/39/commits/d0d7bada6bb4414f3e80df3e357b9bc8af272f69)